docs/source/building.rst

```diff
@@ -54,11 +54,19 @@ Cocotb
 
     .. versionchanged:: 1.6.0 Strip leading and trailing whitespace
 
+    .. deprecated:: 2.0
+
+        :envvar:`TOPLEVEL` is renamed to :envvar:`COCOTB_TOPLEVEL`.
```

Comment:
You aren't actually introducing the new names to the list of variables yet, are you? (Or is the GH client on my cell phone fooling me?)

Reply:
No you're right I just added these messages for each that was renamed, I don't know if that's enough though. My other idea was to introduce the new names in the list and make the old ones point to their description instead.

Reply:
I also think that using the new names and making the old ones be pointers is the best solution.